code/headers/st7735_buffered.hpp

```diff
@@ -21,28 +30,48 @@ namespace r2d2::display {
          * @param reset
          */
         st7735_buffered_c(hwlib::spi_bus &bus, hwlib::pin_out &cs,
-                          hwlib::pin_out &dc, hwlib::pin_out &reset);
+                          hwlib::pin_out &dc, hwlib::pin_out &reset)
+            : st7735_c<Cursor_Count, Display_Size_Width, Display_Size_Height>(bus, cs, dc, reset) 
+        {}
 
         /**
          * @brief Directly write a pixel to the screen
          *
          * @param x
          * @param y
-         * @param data
+         * @param data Data is the color of the pixel
```

Comment:
"data Data" is the color of the pixel? I only see a parameter with "data"

Reply:
Apparently, data is the 16bit color of the pixel. According to the datasheet the RGB values are stored in a 16bit value (5 bits for Red, 6 bits for Green, 5 bits for Blue). This is my best guess for the value, as the owners of the code didn't document it.